js_basics/task3.js

```diff
@@ -0,0 +1,10 @@
+const log = console.log
+
+const drug1 = "Вогнедар"
```

Comment:
use `friend` instead of `drug`. We should think in English while writing the code